src/main/java/mekanism/common/integration/energy/fluxnetworks/FNIntegration.java

```diff
@@ -108,14 +107,15 @@ public long getMaxEnergyStoredL() {
     @Override
     public boolean canExtract() {
         //Mark that we can receive energy if we can insert energy
-        if (!handler.extractEnergy(FloatingLong.ONE, Action.SIMULATE).isZero()) {
+        //todo: is this right? shouldn't it be only if 1 FE works, this is in joules
```

Comment:
This is meant to be in joules, because *technically* the `canReceive` and `canExtract` methods are if we support doing that ever. So this is our attempt at providing an accurate result to those methods.